cypress/integration/testcase.cy.js

```diff
@@ -1,9 +1,118 @@
-const { default: Item } = require("antd/lib/list/Item");
-
 describe("Lotto app", () => {
   beforeEach(() => {
-    cy.visit("https://next-step.github.io/js-lotto/");
+    cy.visit("http://127.0.0.1:5501/index.html");
+  });
+
+  // key event
+  it("input 창에 문자열 입력 시 값이 입력되지 않는다.", () => {
+    const newString = "1000원";
+    cy.get(".lotto-purchase-input").type(`${newString}{enter}`);
+    cy.get(".lotto-purchase-input").should("have.text", "1000");
+  });
+
+  it("input 창에 특수문자 입력 시 값이 입력되지 않는다.", () => {
+    const newPercentSign = "1000%";
+    cy.get(".lotto-purchase-input").type(`${newPercentSign}{enter}`);
+    cy.get(".lotto-purchase-input").should("have.text", "1000");
+  });
+
+  it("엔터 키 입력 시 input 값이 없는 경우 alert 경고를 띄운다", () => {
+    const alertStub = cy.stub();
+    const emptyValue = "";
+
+    cy.on("window:alert", alertStub);
+
+    cy.get(".lotto-purchase-input")
+      .type(`${emptyValue}{enter}`)
+      .then(() => {
+        expect(alertStub).to.be.calledWith("1000원 이상의 값을 입력해주세요.");
+      });
+  });
```

Comment:
여기서 정의한 alertStub 변수는 mockup 함수입니다.
alertStub와 alertStub.getCall(0)의 차이점이 뭔지 알아보면 좋을 것 같아요


아래 링크는 jest 기반 포스팅 글이지만
mockup 에 대한 의미와 tobeCalledWith 구문의 맥락을 이해하기에 좋은 글이니 참고해보세요.

[mocking 이란?](https://www.daleseo.com/jest-fn-spy-on/)

```suggestion
    cy.get(".lotto-purchase-input")
      .type(`${emptyValue}{enter}`)
      .then(() => {
        expect(alertStub.getCall(0)).to.be.calledWith("1000원 이상의 값을 입력해주세요.");
      });
  });
```